TheEthicalHackerCup/Assets/Scenes/AttackMenu.unity

```diff
@@ -217,379 +464,3230 @@ Transform:
+  m_StereoSeparation: 0.022
+--- !u!4 &1124251349
+Transform:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 1124251346}
+  m_LocalRotation: {x: 0.12839712, y: -0, z: -0, w: 0.9917229}
+  m_LocalPosition: {x: 0, y: 4.53, z: -12.44}
+  m_LocalScale: {x: 1, y: 1, z: 1}
+  m_ConstrainProportionsScale: 0
+  m_Children: []
+  m_Father: {fileID: 0}
+  m_RootOrder: 0
+  m_LocalEulerAnglesHint: {x: 14.754, y: 0, z: 0}
+--- !u!1 &1243701245
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 1243701246}
+  - component: {fileID: 1243701250}
+  - component: {fileID: 1243701249}
+  - component: {fileID: 1243701248}
+  - component: {fileID: 1243701247}
+  m_Layer: 5
+  m_Name: TutorialCanvas
+  m_TagString: Untagged
+  m_Icon: {fileID: 0}
+  m_NavMeshLayer: 0
+  m_StaticEditorFlags: 0
+  m_IsActive: 1
+--- !u!224 &1243701246
+RectTransform:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 1243701245}
+  m_LocalRotation: {x: -0, y: -0, z: -0, w: 1}
+  m_LocalPosition: {x: 0, y: 0, z: 0}
+  m_LocalScale: {x: 0.92674977, y: 0.92674977, z: 0.92674977}
+  m_ConstrainProportionsScale: 0
+  m_Children:
+  - {fileID: 345554538}
+  - {fileID: 1111123378}
+  - {fileID: 55543313}
+  - {fileID: 27202200}
+  - {fileID: 65254368}
+  - {fileID: 1837644646}
+  m_Father: {fileID: 835391195}
+  m_RootOrder: 9
+  m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+  m_AnchorMin: {x: 0, y: 0}
+  m_AnchorMax: {x: 0, y: 0}
+  m_AnchoredPosition: {x: 500.90826, y: 239.56482}
+  m_SizeDelta: {x: 1081, y: 517}
+  m_Pivot: {x: 0.5, y: 0.5}
+--- !u!114 &1243701247
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 1243701245}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 49f381a6d1c659045bcb85427fa20e42, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  tutorialActive: 1
+  tutorialCanvas: {fileID: 1243701250}
+  tutorialStates:
+  - tutorialState: {fileID: 1111123377}
+    tutorialText: 'This menu allows you to launch an attack minigame. Each button
+      represents a different attack. '
+    isTutorialExample: 0
+  - tutorialState: {fileID: 55543312}
+    tutorialText: There is a heat bar under each attack button. The heat bar increases
+      after each passive attack but will decrease after each active attack.
+    isTutorialExample: 0
+  - tutorialState: {fileID: 27202199}
+    tutorialText: Try pressing an attack to see how the heat bar works.
+    isTutorialExample: 0
+  - tutorialState: {fileID: 65254367}
+    tutorialText: 
+    isTutorialExample: 1
```

Comment:
isTutorialExample: 1
Be warned that we are revising this functionality of the tutorial because wierd things can heppen when temporary clickable buttons are loaded in during tutorials. 

Right now, I think the plan is replace the temp buttons with arrows pointing to the area of interest